src/compiler/program.ts

```diff
@@ -1378,7 +1378,7 @@ namespace ts {
                 getSourceFile: program.getSourceFile,
                 getSourceFileByPath: program.getSourceFileByPath,
                 getSourceFiles: program.getSourceFiles,
-                getFilesFromNodeModules: () => jsFilesFoundSearchingNodeModules,
+                isSourceFileFromNodeModules: (file: SourceFile) => !!lookUp(sourceFilesFoundSearchingNodeModules, file.path),

```

Comment:
we have `isExternalLibraryImport`, so why not call this `isSourceFileFromExternalLibrary` ?